src/lib389/lib389/cli_conf/backend.py

```diff
@@ -348,24 +348,19 @@ def backend_get_subsuffixes(inst, basedn, log, args):
 def build_node(suffix, be_name, subsuf=False, link=False, replicated=False):
     """Build the UI node for a suffix
     """
-    icon = "glyphicon glyphicon-tree-conifer"
     suffix_type = "suffix"
     if subsuf:
-        icon = "glyphicon glyphicon-leaf"
         suffix_type = "subsuffix"
     if link:
-        icon = "glyphicon glyphicon-link"
         suffix_type = "dblink"
 
     return {
-        "text": suffix,
+        "name": suffix,
```

Comment:
I think it may be that one healcheck test has failed because of the 'cli_conf.backend.py' change.

Reply:
I don't get any unexpected healthcheck errors with this PR.  Plus this code only impacts when you retrieve from the whole tree.  Healthcheck doesn't do any checks on that (not that I know of).

Reply:
I'm talking about the new PR CI - https://github.com/389ds/389-ds-base/pull/4604/checks?check_run_id=1865403669
But maybe there's another reason, I haven't checked manually...

Reply:
The CI tests also pass for me when I run it locally